### ISSUE TITLE
Add Perl6's YAMLish to the matrix

### ISSUE THREAD

**Issue by Leont** [perlpunk/yaml-test-matrix#5]
It's already supported in yam-editor, but not yet in the Matrix. At the recent Perl Toolchain Summit I fixed an important bug that I think previously prevented its inclusion.

**Comment by perlpunk** [comment]
@Leont thanks. My reason for skipping it was that it was too slow on the machine I moved to. I set a timeout for each test at 3 seconds. The plan was to check if a new rakudo build would improve performance. Still haven't had time though...

**Comment by Leont** [comment]
It should precompile, so the first time you load the module may well be quite slow, but all subsequent runs should be reasonably fast.

**Comment by perlpunk** [comment]
yeah, I've seen that in the past, that only the first call of the script is slow, but at some point it started to be slow for every call.

**Comment by perlpunk** [comment]
rakudo 2017.04 seems to be faster, i'm working on including yamlish...
